src/main/java/com/jpr/maintenance/database/model/UserEntity.java

```diff
@@ -3,15 +3,19 @@
 import com.jpr.maintenance.model.Password;
 import com.jpr.maintenance.validation.errors.InputValidationException;
 import com.jpr.maintenance.validation.model.User;
-import lombok.Builder;
-import lombok.Data;
+import lombok.*;
```

Comment:
lets align on settings since my 'optimize imports' will revert this again, i'm fine with either no * at all after 2/5/ etc imports